.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  env: {
+    node: true,
+    es2021: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:cypress/recommended',
```

Comment:
according to recommended configuration - https://www.npmjs.com/package/eslint-plugin-cypress